src/core/server/integration_tests/saved_objects/migrations/group3/multiple_es_nodes.test.ts

```diff
@@ -96,7 +96,7 @@ function createRoot({ logFileName, hosts }: RootConfig) {
 }
 
 // Failing 9.0 version update: https://github.com/elastic/kibana/issues/192624
```

Comment:
comment can be removed

Reply:
Addressed in [`a225053` (#193899)](https://github.com/elastic/kibana/pull/193899/commits/a225053f73c08ccba72e9cd94939e4dc032c5c59)